calculate_trigger.py

```diff
@@ -26,15 +26,38 @@ def allign_fault_to_instruction(address, tbinfo_size, tbinfo_assembler, tbinfo_i
             return asm_addresses[i]
 
 
-def search_for_fault_location(
+def calculate_lifespan_from_start(
     filter_lists,
-    trigger_position,
     fault_address,
-    trigger_occurrences,
     goldenrun_tb_exec,
     goldenrun_tb_info,
+    trigger_occurrences,
+):
+    [idx, instruction_address] = find_fault(
```

Comment:
This was just calculated in `search_for_fault_location()`. It should not be necessary to calculate it again.

Reply:
Well yes but i would need to add two new cache variables that only get used in the case idx < 0. Did not do this to make it easier to understand the code. Also this is a rare edge case, so recalculation is not that costly.

Reply:
Ok, fine for me